### PR TITLE
feat: service routes sync button

### PR DESCRIPTION
**Pull request by yomybaby**
## TL;DR

This PR adds a 'Sync Routes' button to the EndpointDetailPage in the React application. This button allows users to initiate synchronization of routes for a specific endpoint.

## What changed?

- Added a 'Sync Routes' button with associated icons and loading state.
- Created a mutation to handle the route synchronization logic.
- Updated language files to include translations for the new 'Sync Routes' feature.

## How to test?

1. Navigate to the EndpointDetailPage.
2. Verify that the 'Sync Routes' button is present next to the RoutesInfo section.
3. Click the 'Sync Routes' button and observe the success or failure message.

## Why make this change?

This change enhances the usability of the EndpointDetailPage by providing an easy way to synchronize routes directly from the UI.

---

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by yomybaby** [comment]
* **#2471** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2471?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @yomybaby and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: react/src/pages/EndpointDetailPage.tsx [hunk 1]
```diff
@@ -532,7 +548,37 @@ const EndpointDetailPage: React.FC<EndpointDetailPageProps> = () => {
...
+          endpoint?.endpoint_id ? (
+            <Button
+              icon={<SyncOutlined />}
+              loading={mutationToSyncRoutes.isLoading}
+              onClick={() => {
+                endpoint?.endpoint_id &&
+                  mutationToSyncRoutes.mutateAsync(endpoint?.endpoint_id, {
+                    onSuccess: (data) => {
+                      if (data?.success) {
+                        message.success(t('modelService.SyncRoutesSuccess'));
```
Comment (lizable) [inline react/src/pages/EndpointDetailPage.tsx]: I think the translation is missing here.

File: react/src/pages/EndpointDetailPage.tsx [hunk 2]
```diff
@@ -532,7 +548,37 @@ const EndpointDetailPage: React.FC<EndpointDetailPageProps> = () => {
...
+                        startRefetchTransition(() => {
+                          updateFetchKey();
+                        });
+                      } else {
```
Comment (lizable) [inline react/src/pages/EndpointDetailPage.tsx:567]: Here as well.

**Review by lizable — CHANGES_REQUESTED** [review]
Some translations are missing. Except that, It works fine.

**Review by lizable — APPROVED** [review]
By suggestion of @YEONFEEL96, we polished up the message for route synchornization to match with the action.

**Pull request merged**